lib/saml_idp/request.rb

```diff
@@ -102,6 +103,12 @@ def requested_aal_authn_context
       end.first
     end
 
+    def requested_vtr_authn_context
+      requested_authn_contexts.select do |classref|
+        VTR_REGEXP.match?(classref)
+      end.first
```

Comment:
I think it might be preferable to require the array as in OIDC. If a SAML partner wants to support multiple types of requirements in the response (e.g. IALMAX), that wouldn't be possible with this. We end up converting to an array anyway: https://github.com/18F/identity-idp/pull/10178/files#diff-ae40aeed74a7550dc38228836ae7badab4a5c01dd3dac01f9b64abd8556697c6R27-R29